### PR TITLE
add timestamps attributes to false

### PR DESCRIPTION
**Pull request by rorikurniadi**
Hey, I need CRUD Province, City, District, and Village. so I added `$timestamps` for support that.

### CODE REVIEW

File: src/migrations/2016_08_03_072729_create_provinces_table.php [hunk 1]
```diff
@@ -15,7 +15,7 @@ public function up()
     {
         Schema::create(config('laravolt.indonesia.table_prefix') . 'provinces', function(Blueprint $table)
         {
-            $table->char('id', 2);
+            $table->char('id', 3);
```
Comment (uyab) [inline src/migrations/2016_08_03_072729_create_provinces_table.php]: Why you change this?

Comment (rorikurniadi) [reply src/migrations/2016_08_03_072729_create_provinces_table.php]: I've spec add more Province data. and it reach more than 99. so I modify that for support the spec.

Comment (uyab) [reply src/migrations/2016_08_03_072729_create_provinces_table.php]: Please make separate PR for those

Comment (rorikurniadi) [reply src/migrations/2016_08_03_072729_create_provinces_table.php]: ok cool @uyab!

**Pull request merged**